master/internal/scheduler/filterable_view.go

```diff
@@ -45,56 +45,56 @@ func (v *FilterableView) Update(cluster *Cluster) (ViewSnapshot, bool) {
 }
 
 func (v *FilterableView) updateTasks(cluster *Cluster) bool {
-	updateMade := false
-	tasks := make(map[TaskID]*TaskSummary)
+	newTasks := make(map[TaskID]*TaskSummary)
```

Comment:
non-blocking: `newTasks` to me implies to me that this would be something like `newTasks = allTasks - oldTasks`

Reply:
That's fair. My thought process was that `tasks` vs. `v.tasks` was a fairly subtle difference in naming; by `newTasks` I mean "the new version of the `tasks` map".